jobs/config.json

```diff
@@ -9412,6 +9412,28 @@
       "UNKNOWN"
     ]
   },
+  "pull-kubernetes-e2e-gce-ipvs": {
+    "args": [
+      "--build",
+      "--cluster=",
+      "--env-file=jobs/platform/gce.env",
+      "--env-file=jobs/env/pull-kubernetes-e2e.env",
+      "--env-file=jobs/env/pull-kubernetes-e2e-gce-ipvs.env",
+      "--extract=local",
+      "--gcp-node-image=gci",
+      "--gcp-zone=us-central1-f",
+      "--ginkgo-parallel=30",
+      "--mode=docker",
```

Comment:
`--mode=docker` does not work on Prow since kubernetes doesn't really support running docker commands from your pods, please use `--build=bazel` and drop the `--mode` flag